### PR TITLE
Layover: add theme

### PR DESCRIPTION
**Pull request by henriqueiamarino**
Layover displays large titles and excerpts that scroll over an image on the Homepage and neat single pages for users who want their blogging to be simple.
[Demo site](https://layover.mystagingwebsite.com/)

screenshot

### CODE REVIEW

**Comment by henriqueiamarino** [comment]
The `background-image` inside the Cover in Home is not showing, and I can't fix it.
I ask the reviewer to check it, please.

**Comment by jasmussen** [comment]
**General visual observations**

This is a favorite of mine, looks fantastic.

It's a bit hard to review fully, though, as I'm also not seeing the background image you're referring to. But I'm not actually seeing any Cover block at all in the Home template:

Screenshot 2024-08-07 at 10 44 05

Feel free to hit me up in chat and we can review sync if need be, maybe screenshare, see where it might be missing or what I should check.

Other than that, virtually everything looks great. Good on mobile, no glitches, templates nice. 

Style variations are a bit bare. The muted colors are nice, but I'm missing both some font alternatives, and perhaps a light/dark monochrome styles. Not blockers, mainly food for thought.

I did notice one awkwardly named font:

Screenshot 2024-08-07 at 10 46 21

In fact, there are quite a lot of fonts in the font folder, but in testing the templates, it looks like only a single sans-serif font is used. Maybe you did add fonts for the style variations, but they're not working fully as intended? Something to look at.

**File and readme reviews**

Readme looks good.

**Summary**

Great theme, will be a nice one to land. We need to fix the missing image issue, and figure out what's up with the fonts. Other than that, this one is close.

**Comment by henriqueiamarino** [comment]
Thanks, @jasmussen. I fixed as much as I could and improved the style variations. I need help fixing the home template, though. CBT plugin replaces the image file with a URL from the demo site and fails to load it.

File: layover/patterns/home.php [hunk 1]
```diff
@@ -6,8 +6,8 @@
...
-<div class="wp-block-cover has-parallax" style="margin-top:0rem;margin-bottom:0rem;padding-top:0rem;padding-right:0rem;padding-bottom:0rem;padding-left:0rem"><span aria-hidden="true" class="wp-block-cover__background has-background-dim-0 has-background-dim" style="background-color:#212121"></span><div class="wp-block-cover__image-background  has-parallax" style="background-position:50% 50%;background-image:url(http://localhost.local/wp-content/uploads/2024/08/layover_background.jpg)"></div><div class="wp-block-cover__inner-container"><!-- wp:template-part {"slug":"header","align":"wide"} /-->
+<!-- wp:cover {"url":"<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/layover_background.jpg","hasParallax":true,"dimRatio":0,"customOverlayColor":"#212121","isUserOverlayColor":true,"contentPosition":"center center","style":{"spacing":{"margin":{"top":"0rem","bottom":"0rem"},"padding":{"top":"0rem","right":"0rem","bottom":"0rem","left":"0rem"}}},"layout":{"type":"constrained"}} -->
```
Comment (jasmussen) [inline layover/patterns/home.php:9]: I pushed a change manually to fix this issue. Not sure why CBT didn't fix this: did we do something wrong in the export process, @matiasbenedetto? The image landed in the right directory, it just wasn't attached here.

**Comment by jasmussen** [comment]
With the latest fix, I think this is ready to merge! Nice work.

**Pull request merged**